metadata-protobuf/proto/WorkingGroups.proto

```diff
@@ -57,3 +57,14 @@ message WorkingGroupMetadataAction {
     RemoveUpcomingOpening remove_upcoming_opening = 3;
   }
 }
+
+message ModeratePost {
+  required uint64 post_id = 1;
+  optional string rationale = 2;
```

Comment:
I think `rationale` should also be `required`, since in `PostModeratedEvent` [entity](https://github.com/thesan/joystream/blob/096ae678145fb249bf024956abdb3be82d1fb9d9/query-node/schemas/forumEvents.graphql#L288) it is defined as non-nullable field. So maybe it's consistent to have `rationale` in protobuf schema as required too.

Reply:
Right! I missed that